streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/impl/KubernetesApplicationClientV2.scala

```diff
@@ -103,12 +103,15 @@ object KubernetesApplicationClientV2 extends KubernetesClientV2Trait with Logger
       .filter(str => StringUtils.isNotBlank(str))
       .getOrElse(return Left("Flink base image should not be empty"))
 
-    val ingress = submitReq.k8sSubmitParam.ingressDefinition
+    val ingress = Option(submitReq.k8sSubmitParam.ingressDefinition)
+      .getOrElse(return Left("Ingress should not be empty"))
 
-    val imagePullPolicy = flinkConfObj
-      .getOption(KubernetesConfigOptions.CONTAINER_IMAGE_PULL_POLICY)
-      .map(_.toString)
-      .orElse(submitReq.k8sSubmitParam.imagePullPolicy)
+    val imagePullPolicy = Option(
```

Comment:
imagePullPolicy is a Option[String] here which is not mandatory to have a value.